executor/executor_test.go

```diff
@@ -5741,16 +5744,10 @@ func TestAdmin(t *testing.T) {
 	// Test for reverse scan get history ddl jobs when ddl history jobs queue has multiple regions.
 	txn, err = store.Begin()
 	require.NoError(t, err)
-	historyJobs, err = ddl.GetHistoryDDLJobs(txn, 20)
+	historyJobs, err = ddl.GetHistoryDDLJobs(testkit.NewTestKit(t, store).Session(), txn, 20)
 	require.NoError(t, err)
 
-	// Split region for history ddl job queues.
-	m := meta.NewMeta(txn)
-	startKey := meta.DDLJobHistoryKey(m, 0)
-	endKey := meta.DDLJobHistoryKey(m, historyJobs[0].ID)
-	cluster.SplitKeys(startKey, endKey, int(historyJobs[0].ID/5))
```

Comment:
Why is it removed?

Reply:
history ddl job queue is no longer exists, it will always empty